index.html

```diff
@@ -3,14 +3,90 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
```

Comment:
We seem to be missing the required viewport meta tag here. Don't forget to add that in!